### ISSUE TITLE
Iterative polling to update the frontend

### ISSUE THREAD

**Issue by DanielKraft** [hochschule-darmstadt/einander-helfen#270]
**Reason (Why?)**
At the current time, it may happen that after updating the frontend, the old version is still in the browser's cache.

**Solution (What?)**
Automatic updating of the frontend should be implemented.

**Out of scope**

**Relation to other Issues**

**Acceptance criteria**

**Comment by BernhardHumm** [comment]
The issue should be renamed to “automatic refresh of browser cache for new releases”

 

Iterative polling is just one solution and possibly not the best (e.g. comparing version numbers could be more efficient)

 

 

 

------------------------------------

 

Von: Daniel Kraft <notifications@github.com> 
Gesendet: Donnerstag, 21. Januar 2021 10:40
An: hochschule-darmstadt/einander-helfen <einander-helfen@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [hochschule-darmstadt/einander-helfen] Iterative polling to update the frontend (#270)

 

Reason (Why?)
At the current time, it may happen that after updating the frontend, the old version is still in the browser's cache.

Solution (What?)
Automatic updating of the frontend should be implemented.

Out of scope

Relation to other Issues

Acceptance criteria

**Comment by mhendlich** [comment]
I dont think this is possible for us without employing some complex release management mechanism, which would just be overkill here.
Manual refreshes will always work fine as Webpack is building cache-busting bundles by appending a modules hash to its file name.
